source/about/maximize-microsoft-investment.rst

```diff
@@ -11,4 +11,5 @@ Supplement existing investments in Microsoft Teams, M365, and Entra ID with Matt
 * :doc:`Mattermost for Microsoft Teams Messaging </integrate/microsoft-teams-interoperability>` - Integrate Microsoft Teams to securely flow messaging from Teams into Mattermost channels and share information and requests across environments in real time.
```

Comment:
I wasn't aware of this page until now: can we update this text to reflect the notifications flow we're shipping with v10?

Reply:
@lieut-data - This is a brand new landing page for Microsoft integrations for Mattermost. What about the text above isn't aligned with the new notifications flow? I ask because I took this messaging directly from our latest Marketing materials.

Reply:
We don't offer anything that flows messaging from Teams into Mattermost channels -- unless that's a blurry way of referring to the chat notification feature in v10? This reads more like 2-way sync.